src/Codegen/Codegen.hs

```diff
@@ -669,8 +670,9 @@ genExpr (I.Match s as t) = do
 
   (cases, blks) <- bimap concat concat . unzip <$> mapM genArm as
   return (val, sStms ++ assignScrut ++ switch cases ++ blks ++ joinStm)
-genExpr I.Lambda{}      = fail "Cannot handle lambdas"
-genExpr (I.Prim p es t) = genPrim p es t
+genExpr I.Lambda{}        = fail "Cannot handle lambdas"
+genExpr (I.Prim p es t  ) = genPrim p es t
+genExpr (I.Exception _ _) = error "cannot happen" -- for me TODO
```

Comment:
For now, can you generate an `ssm_throw()`? In particular, use [`throw`](https://github.com/ssm-lang/sslang/blob/main/src/Codegen/LibSSM.hs#L99) with [`INTERNAL_ERROR`](https://github.com/ssm-lang/sslang/blob/main/src/Codegen/LibSSM.hs#L85)